### PR TITLE
[TW-593] Clarified pre-request and test scripts at the folder level

### PR DESCRIPTION
**Pull request by MatthewPinsker-postman**
In multiple topics, clarified that scripts at the folder level will run before or after requests that are direct children of the folder. Also, fixed a few Vale suggestions.

### CODE REVIEW

File: src/pages/docs/writing-scripts/intro-to-scripts.md [hunk 1]
```diff
@@ -61,7 +61,7 @@ warning: false
 
 ## Scripts in Postman
 
-Postman contains a powerful runtime based on Node.js that allows you to add dynamic behavior to requests and collections. This allows you to write API tests, build requests that can contain dynamic parameters, pass data between requests, and a lot more. You can add JavaScript code to execute during 2 events in the flow:
+Postman has a powerful runtime based on Node.js that allows you to add dynamic behavior to requests and collections. This allows you to write API tests, build requests that can contain dynamic parameters, pass data between requests, and a lot more. You can add JavaScript code to execute during 2 events in the flow:
```
Comment (jkonrath-postman) [inline src/pages/docs/writing-scripts/intro-to-scripts.md:64]:
```suggestion
Postman has a powerful runtime based on Node.js that allows you to add dynamic behavior to requests and collections. This allows you to write API tests, build requests that can contain dynamic parameters, pass data between requests, and a lot more. You can add JavaScript code to execute during two events in the flow:
```

**Review by jkonrath-postman — APPROVED** [review]
LGTM!

**Review by giridharvc7 — APPROVED** [review]
LGTM

**Review by mark-anthony-postman — APPROVED** [review]
LGTM! 💎 ✨

**Pull request merged**